test/trigger-circleci-workflows.test.ts

```diff
@@ -8,9 +8,18 @@ nock.disableNetConnect();
 
 const EXAMPLE_CONFIG = `
 labels_to_circle_params:
-  ci/binaries: run_binaries_tests
-  ci/bleh: run_bleh_tests
-  ci/foo: run_foo_tests
+  ci/binaries:
+    parameter: run_binaries_tests
+    default_true_on:
```

Comment:
Something that seems a bit obscured in this new configuration format is that `default_true_on` seems to be forcing a 1-1 correspondence between labels and parameters. (because `default_true_on` is acting on the *parameter*, not the *label*, but you've put it in the config block for `ci/binaries`). But as you've written this configuration format, there's nothing stopping you from repeating a parameter in multiple times (and indeed, you could easily imagine a generalization to parameter which accepts multiple parameters, so that a single label lets you trigger multiple things). And now `default_true_on` is acting on labels, not parameters, which seems a bit goofy: really this is an orthogonal matter.

This is mostly just an aesthetic/UX problem, since you can express everything in this format as you could have done otherwise by just making sure you have labels per parameter. However, it will be pretty annoying to fix the config format once you deploy this, so I'd suggest spending a little time making it nice right now)

Reply:
Oh so you're suggesting flipping the relationship?

So instead of being labels that map to parameters it's rather parameters that map to labels?

That could work, it'd just take some refactoring.